src/PrpShop/QPlasmaUtils.cpp

```diff
@@ -606,6 +607,11 @@ std::vector<short> pqGetValidKOTypes()
     return std::vector<short>(s_typeList, s_typeList + s_numTypes);
 }
 
+bool pqIsValidKOType(short objType) {
+    auto valid_types = pqGetValidKOTypes();
```

Comment:
```suggestion
    const auto valid_types = pqGetValidKOTypes();
```

Reply:
Is there a benefit to this aside from correctly conveying intention?

Reply:
The benefit is that the `find` call will use `const_iterator`s instead of mutable iterators, which may allow the compiler to optimize away duplicate calls...  It may not matter here anyway since the vector is built at run-time rather than statically, but yes, it also helps convey intention ;)

Reply:
Ah, thanks.  Good to know.  I was wondering if `const_iterator` was involved.